features/snippets.feature

```diff
@@ -48,7 +48,7 @@ Feature: undefined step snippets
     When I run feature suite
     Then the undefined step snippets should be:
       """
-      func iSendRequestToWith(arg1, arg2 string, arg3 *messages.PickleStepArgument_PickleTable) error {
+      func iSendRequestToWith(arg1, arg2 string, arg3 *godog.Table) error {
```

Comment:
@lonnblad Is this test not enough?

Reply:
it is for `*godog.Table`, but I would like a test for `*godog.DocString` as well